engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -67,25 +67,26 @@ namespace dmGameSystem
 
     struct SpriteComponent
     {
-        dmGameObject::HInstance     m_Instance;
+        Matrix4                     m_World;
```

Comment:
Classic strategy: Sort items on struct size.
This is the _only_ optimization here.

Caveat: it may be annoying as two related members are put in different places.